src/modules/emote_menu/index.js

```diff
@@ -1,3 +1,16 @@
-import EmoteMenu from './EmoteMenu.jsx';
+import TwitchEmoteMenu from './twitch/EmoteMenu.jsx';
+import YoutubeEmoteMenu from './youtube/EmoteMenu.jsx';
+
+let EmoteMenu = null;
+
+switch (window.location.hostname) {
+  case 'www.youtube.com':
```

Comment:
this should not be derived in a module. instead, there should be some constant value determining the entrypoint